### PR TITLE
Added test-cases for Active Directory UserGroups

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

File: tests/foreman/ui/test_adusergroups.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    def test_create_ldap_auth_withAD(self):
+        """@Test: Create LDAP authentication with AD
+
+        @Feature: Active Directory - create LDAP AD
+
+        @steps:
+
+        1. Create a new LDAP Auth source with AD
+
+        2. Fill in all the fields appropriately for AD
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_adusergroups.py]: Can you drop the space between each list item?

    1. Create a new LDAP Auth source with AD
    2. Fill in all the fields appropriately for AD

File: tests/foreman/ui/test_adusergroups.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        @Steps:
+
+        1. Create an UserGroup.
+
+        2. Assign some foreman roles to UserGroup.
+
+        3. create an External AD UserGroup as per the usergroup name in AD
+
+        4. Repeat steps 1) and 2) and provide the same usergroup name for
+        step 3)
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_adusergroups.py]: Make sure to evenly indent all text that follows a list marker.

    4. Repeat steps 1) and 2) and provide the same usergroup name for
       step 3)

File: tests/foreman/ui/test_adusergroups.py (hunk 1 above)
Comment (Ichimonji10) [inline tests/foreman/ui/test_adusergroups.py]: Comment applies throughout file.

**Comment by Ichimonji10** [comment]
ACK pending comments.

**Comment by elyezer** [comment]
Please add `@stubbed()` decorator to each stub.

ACK, pending comment.

**Pull request merged**